process/charter.html

```diff
@@ -400,7 +400,7 @@ <h3 id="managing-changes">5.3 Managing Changes to Charters</h3>
 </li>
 </ol>
 
-<h2>6. <a id="director-decision" name="director-decision">
+<h2 id="w3c-decision">6. <a id="director-decision">
```

Comment:
Perhaps this was intended:

```suggestion
<h2 id="w3c-decision">6. <a id="w3c-decision">
```

Aside: The document needs a passthrough with headings id and/or anchor id

Reply:
I intentionally left the #director-decision fragment to avoid breaking inbound links. If I'd been adding it, I'd have used `<a id="director-decision"></a>` just before the heading, but it didn't seem important enough to move around the existing `<a>`. No objections if the Team prefers another arrangement, or to remove the old ID.

Reply:
And, there shouldn't be duplicate `id` values any way.. which was one of the reasons why I mentioned the document needs a pass.